src/esn.calendar.libs/i18n/en.json

```diff
@@ -189,7 +189,7 @@
   "Organizer": "Organizer",
   "Organizers": "Organizers",
   "Owner": "Owner",
-  "Participation updated: %s": "Participation updated: {{ participation }}",
+  "Participation updated to:": "Participation updated to: ",
```

Comment:
```suggestion
  "Participation updated to:": "Participation updated to",
```

Reply:
it was the suggestion of @huy-ta

Reply:
I meant the value is not correct like you did for other langs?